SensioLabs/Security/Crawler/CurlCrawler.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the SensioLabs Security Checker.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SensioLabs\Security\Crawler;
+
+use SensioLabs\Security\Exception\RuntimeException;
+
+class CurlCrawler extends BaseCrawler
```

Comment:
well, the SecurityChecker class is not really internal. It is the main API of the SDK. These classes on the other hand should be marked as internal IMO to inform people it makes no sense using them as extension points (they are here to support multiple way to perform the HTTP request)